eng/pipelines/templates/scripts/verify-metadata-location.yml

```diff
@@ -0,0 +1,12 @@
+# This check fails when there's file in `src/SDKs/_metadata/` directory
```

Comment:
I wouldn't do this as a template. I'd just do it inline. the mgmt.yml.

If you do decide to keep the template can you please put it under `eng/pipelines/templates/steps` scripts folder is intended for actual scripts not yml files.

Reply:
Great! Moved the scripts inline.